scripts/write-env-example-file.ts

```diff
@@ -1,17 +1,16 @@
 import * as fs from 'fs';
 import * as api3Chains from '@api3/chains';
 
-const chainsEnvVars = [
-  'MNEMONIC',
-  'RPC_URL_LOCALHOST',
-  ...api3Chains.CHAINS.map((chain) => {
-    return `RPC_URL_${chain.alias.replace(/-/g, '_').toUpperCase()}`;
-  }),
-];
+const chainsEnvVars = ['localhost', ...api3Chains.CHAINS.map((chain) => chain.alias)].map(
+  (chainAlias) => `FUNDER_RPC_URL_${chainAlias.replace(/-/g, '_').toUpperCase()}`
+);
 
 fs.writeFileSync(
-  '.env.example',
-  chainsEnvVars.reduce((fileContents: string, envVariableName: string) => {
-    return fileContents + `${envVariableName}=""\n`;
-  }, '')
+  'example.env',
+  [...chainsEnvVars, ...api3Chains.hardhatConfig.getEnvVariableNames()].reduce(
```

Comment:
This adds the etherscan env vars, right?

Reply:
Yeah, `getEnvVariableNames()` includes them and `MNEMONIC`